etc/config/c++.amazon.properties

```diff
@@ -148,15 +148,12 @@ compiler.clang_concepts.options=-std=c++2a -Xclang -fconcepts-ts -stdlib=libc++
 compiler.clang_concepts.notification=Highly experimental compiler. Bug reports welcomed at <a href="https://github.com/saarraz/clang-concepts/issues" target="_blank" rel="noopener noreferrer">github.com/saarraz/clang-concepts/issues <sup><small class="glyphicon glyphicon-new-window opens-new-window" title="Opens in a new window"></small></sup></a>
 
 # Clang for RISC-V
-group.rvclang.compilers=rv32clang:rv64clang
+group.rvclang.compilers=rv32clang
 group.rvclang.groupName=Clang RISC-V
 group.rvclang.supportsBinary=false
 compiler.rv32clang.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
-compiler.rv32clang.name=Clang RISC-V (32 bit)
+compiler.rv32clang.name=Clang riscv32 (trunk)
```

Comment:
One last nitpick: all the other architectures are listed first in their compiler name ("ARM gcc" ... "x86 clang" ... "MSP430 gcc" etc) ... although their group name is "Compiler Arch".

Can you change the name here to be "RISC-V clang (trunk)" (assuming the capitalisation is correct) and then I'll merge. We need a better solution to this in general. the dropdown is terrible already (see #137 #257 etc), but at least consistency with the (bad) current setup is better than none.

Reply:
Argh, it would need the 32-bit indicator too I guess, "riscv32 clang (trunk)" maybe?